swag-proxy/aam-prod-3/docker-compose.yml

```diff
@@ -18,7 +18,6 @@ services:
       - EXTRA_DOMAINS=*.aam-digital.net,*.aam-digital.app,*.aam-digital.com
     volumes:
       - ./config/dns-conf/hetzner.ini:/config/dns-conf/hetzner.ini
-      - ./config/nginx/proxy-confs/aam-prod-3.subdomain.conf:/config/nginx/proxy-confs/aam-prod-3.subdomain.conf
```

Comment:
why are two conf files deleted in this PR but only one removed from a docker-compose? Was the other not used?